src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/two_inputs_and_outputs.cpp

```diff
@@ -10,6 +10,10 @@ namespace test {
 namespace snippets {
 namespace {
 
+// todo: Remove the architecture constraint after isa sve_128 being supported. Because for now ARM Snippets only support isa asimd,
+// but dnnl injector jit_uni_eltwise_injector_f32 requires isa being at least sve_128. So dnnl emitters used by these test cases
+// are not supported for ARM architecture now.
+#if defined(OPENVINO_ARCH_X86_64)
```

Comment:
Is it possible just to disable these tests in `skip_test_config`?

Reply:
Applied. Thanks Alexandra!